packages/terra-collapsible-menu-view/src/CollapsibleMenuViewItemGroup.jsx

```diff
@@ -107,7 +107,7 @@ class CollapsibleMenuViewItemGroup extends React.Component {
       }
       return (
         <li role="none">
-          <div {...customProps} role="group">
+          <div {...customProps} aria-label={ariaLabel} role="group">
```

Comment:
Same here the example updated with aria-label with this PR will work without this change. 
`aria-label={ariaLabel}`